anomalib/utils/loggers/console.py

```diff
@@ -0,0 +1,110 @@
+"""Console Logger."""
+
+# Original Copyright (c) OpenMMLab.
```

Comment:
I think this should say Anomalib. Also, should we change the date to 2022?

Reply:
no, the code has been modified based on OpenMMLab, so we give the credit.